README.md

```diff
@@ -1,2 +1,367 @@
 # ocp-virt-validation-checkup
-Scripts and utilities for running the OpenShift Virtualization validation checkup
+This repository provides the necessary scripts and utilities to run a validation checkup of OpenShift Virtualization inside a customer environment.
+
+## Objectives
+* Run a stable subset of functional tests of [KubeVirt](https://github.com/kubevirt/kubevirt)'s main SIGs: Compute, Network, Storage, as well as [SSP](https://github.com/kubevirt/ssp-operator) test suite. These are also known as Tier-1 test suites.
+* Self-sustained, unified package that contains all of the required code to run the validation checkup. The test code is fully aligned with the extact OpenShift Virtualization build installed on the cluster under test. The test suites are being run by a Job object, which runs to a completion.
+* Easily run the test suites, with optional tweaks.
+* Easily get and browse the validation checkup results. The test suite results, logs and artifacts are being stored in a PersistentVolumeClaim. A summary of the results is being stored to a ConfigMap.
```

Comment:
Not sure about the word "Easily" here. Sound like an ad to me (:

Maybe something like "Configuring and Running the test suites, should be a s simple as possible" - this one is not great either, but something like that.

Reply:
I rephrased it to "...in a streamlined way", and "...with little effort".
is it better?